pagestyles/styles.js

```diff
@@ -1,4 +1,22 @@
 import styled from "styled-components";
+import Modal from "react-modal";
+const ModalBody = styled.body`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
```

Comment:
I am glad, that you don't use this ModalBody anywhere in your project. Because you cannot have another body inside your ``<body>``. Since you don't use it, please remove it.